modules/visual_script/visual_script_builtin_funcs.cpp

```diff
@@ -826,6 +835,24 @@ void VisualScriptBuiltinFunc::exec_func(BuiltinFunc p_func,const Variant** p_inp
 			VALIDATE_ARG_NUM(0);
 			*r_return=Math::db2linear(*p_inputs[0]);
 		} break;
+		case VisualScriptBuiltinFunc::MATH_RAD2VEC: {
+
+			VALIDATE_ARG_NUM(0);
+			float v2x, v2y;
+			v2x = Math::cos(*p_inputs[0]);
+			v2y = Math::sin(*p_inputs[0]);
+			Vector2 v (v2x,v2y);
+			*r_return = v;
```

Comment:
Same as above, `r_return = Vector2(Math::cos(..), Math::sin(..))` might be way shorter and maybe a bit simpler.